content/services/csv/powerdev_graduated_open_source_projects.csv

```diff
@@ -0,0 +1,15 @@
+"Alberta Speculation","Investigate multi-threaded speculation of alternative paths of execution in a sequential execution"
+"Diffusion magnetic resonance imaging of human brain","Drive diffusion basis spectrum imaging (DBSI) for clinical applications and large scale trials for a diffusion MRI modality which can increase the magnetic resonance imaging specificity for neurodegeneration diseases"
+"Drupal Testbots","Powered the testing and review of code contribution for Drupal 8 (qa.drupal.org)"
+"GCC","Contributed to improving support of various open source projects in the GCC family on POWER machines; began by tackling IBM Linux Technology Center items on bountysource.com"
+"gdb","Worked on adding fast tracepoint support for ppc/ppc64/ppc64le (https://sourceware.org/bugzilla/show_bug.cgi?id=17221)"
```

Comment:
Instead of listing the actual URL in the text, I recommend you just make it a normal link tied to a word. Otherwise it makes it really long in the description.